### PR TITLE
ENH: add SUNY buffalo to institutions db

### PR DESCRIPTION
**Pull request by sbanerjee23**
Added SUNY Buffalo and Dept of Chemical and Biological Engineering for parHAM

### CODE REVIEW

File: db/institutions.yml [hunk 1]
```diff
@@ -149,3 +149,11 @@ ucopenhagen:
...
   state: ''
   zip: ''
+sunybuffalo:
+  aka:
+  city: Buffalo
+  country: USA
+  departments: {cheme: Department of Chemical and Biological Engineering}
+  name: The State University of New York at Buffalo
+  state: ''
+  zip: ''
```
Comment (CJ-Wright) [inline db/institutions.yml]: I think this needs a newline here.

File: db/institutions.yml [hunk 2]
```diff
@@ -149,3 +149,11 @@ ucopenhagen:
   name: University of Copenhagen
   state: ''
   zip: ''
+sunybuffalo:
+  aka:
```
Comment (sbillinge) [inline db/institutions.yml]: how about SUNY-Buffalo in the aka?

File: db/institutions.yml [hunk 3]
```diff
@@ -150,10 +150,10 @@ ucopenhagen:
   state: ''
   zip: ''
 sunybuffalo:
-  aka:
+  aka: [SUNY Buffalo]
```
Comment (sbillinge) [inline db/institutions.yml]: I think this should be SUNY-Buffalo, or we should add this also

**Comment by sbanerjee23** [comment]
Closes #4

**Pull request merged**